### PR TITLE
LSNBLDR-830 Embedded Announcements and Forum posts in Lesson page don…

### PR DESCRIPTION
**Pull request by Johnson-Mickey**
…'t render correctly.

### CODE REVIEW

**Comment by Johnson-Mickey** [comment]
I erroneously submitted this with LSNBLDR-769 in the title, it should have been 830.

File: lessonbuilder/tool/src/webapp/css/Simplepagetool.css [hunk 1]
```diff
@@ -1076,6 +1076,7 @@ img {
  border-radius: 0px 0px 10px 10px;
  border: solid #999 1px;
  border-top-style: none;
+ display: inherit !important;
```
Comment (ottenhoff) [inline lessonbuilder/tool/src/webapp/css/Simplepagetool.css]: Can we avoid !important somehow?

**Comment by Johnson-Mickey** [comment]
@ottenhoff I have made an amendment to try and avoid !important, please check it out when you have a moment.

**Comment by ern** [comment]
@Johnson-Mickey can you amend your commit and update with the correct JIRA issue?

**Comment by ern** [comment]
if you need assistance in doing so ask on the sakaidev slack channel.

**Comment by Johnson-Mickey** [comment]
@ern Amended.

**Pull request merged**